lib/appeals_status/service.rb

```diff
@@ -31,8 +46,45 @@ def try_mocks(mock_responses_method, user, response_class)
       nil
     end
 
+    def format_response_reasonably(response)
+      response = response.deep_symbolize_keys
+      format_hearings_reasonably(response)
+      response[:data].each do |appeal|
+        appeal[:events] = appeal[:attributes].delete(:events)
+        appeal[:attributes].each do |k, v|
+          appeal[k] = v
+        end
+        appeal.delete(:attributes)
+        appeal.delete(:relationships)
+      end
+      response[:data]
+    end
```

Comment:
Yep, it's because the mock service doesn't use the middleware. I think I'd rather keep it this way where I can test the mock thing in the same way as the real one, but I don't feel strongly about that. I also think the middleware would make more sense if we were going to reuse this logic, but I'm not aware that we'll need to.